src/App.js

```diff
@@ -19,11 +19,7 @@ export default function App() {
   };
 
   return (
-    <FavouriteProvider 
-      value={{
-        favouritePokens: favourites, 
-        updateFavoritePokemons: updateFavoritePokemons
-      }}>
+    <FavouriteProvider>
```

Comment:
```suggestion
    <Provider>
```

Creo que este nombre va mejor